gk/main.c

```diff
@@ -1217,6 +1416,14 @@ alloc_gk_conf(void)
 	return rte_calloc("gk_config", 1, sizeof(struct gk_config), 0);
 }
 
+void
+set_gk_request_timeout(unsigned int request_timeout_sec,
+	struct gk_config *gk_conf)
+{
+	gk_conf->request_timeout_cycles =
+		(uint64_t)request_timeout_sec * rte_get_tsc_hz();
```

Comment:
Actually, the return of `rte_get_tsc_hz()` is `uint64_t`, so we can drop `(uint64_t)` in the expression above. It was my fault to suggest it in the first place.